### ISSUE TITLE
Allow converting multiple editor cursors into tab stops

### ISSUE THREAD

**Issue by Mtn-View** [microsoft/vscode#235528]
I would like to be able to turn multiple editor cursors into tab stops. 

A lot of the time, I want to edit the same value on an array of objects, but set each one to a different value, or something like that. So, for example, if I have an array of objects all with the `name` property, I might want to select `name: ''` on the first one, hit `CTRL + D` a bunch to select them all, and then hit another keyboard shortcut (or command), either turn them all into tab stops that I can then tab between, or have a command like "Focus Next Cursor" and "Focus Previous Cursor" that changes which cursor I'm typing into.

**Comment by daytonlowell** [comment]
Yeah, as an avid multi-cursor user, I too would really appreciate this feature.
